nixos/modules/services/continuous-integration/github-runner/query_registration_token.py

```diff
@@ -0,0 +1,85 @@
+import sys, json
```

Comment:
```suggestion
import json
```

`sys` import is unused.